### PR TITLE
Fixed Issue 138

### PR DESCRIPTION
**Pull request by Vesiix**
This implements Michelle Crawley's fix for the RegisterParticipantIntake stored procedure.

### CODE REVIEW

**Comment by Vesiix** [comment]
Still needs Steve's comment implemented.

**Review by marcosb1 — APPROVED** [review]

File: PROCEDURES.sql [hunk 1]
```diff
@@ -171,7 +171,7 @@ $BODY$
         IF FOUND THEN
             pID := (SELECT Participants.participantID FROM Participants WHERE Participants.participantID = intakeParticipantID);
             UPDATE Participants
-            SET Participants.race = intakeParticipantRace,
+            SET Participants.race = intakeParticipantRace AND
```
Comment (blackjack26) [inline PROCEDURES.sql]: This should probably still be a comma

Comment (blackjack26) [inline PROCEDURES.sql]: This should still be a comma and not `AND`.

**Review by blackjack26 — CHANGES_REQUESTED** [review]

File: PROCEDURES.sql (hunk 1 above)
Comment (Vesiix) [reply PROCEDURES.sql]: Yea, I accidentally reverted it in my second commit. Good catch, it's fixed now!

**Review by jrands — APPROVED** [review]

**Pull request merged**